docs/diagrams/StarCommandSequenceDiagram.puml

```diff
@@ -30,34 +35,34 @@ deactivate StarCommandParser
 AddressBookParser -> StarCommandParser : parse("1 s/1")
```

Comment:
Arguments passed from AddressBookParser to StarCommandParser include a whitespace in front, hence parse(" 1 s/1") would be more accurate.

Reply:
okay